content/docs/rfcs/16/README.md

```diff
@@ -451,14 +452,15 @@ This method is part of the `store` API and the specific resources to retrieve ar
 
 #### Request
 
-```curl -d '{"jsonrpc":"2.0","id":"id","method":"get_waku_v2_store_v1_messages", "params":[["/waku/2/default-content/proto"]]}' --header "Content-Type: application/json" http://localhost:8545```
+```curl -d '{"jsonrpc":"2.0","id":"id","method":"get_waku_v2_store_v1_messages", "params":["", ["/waku/2/default-content/proto"]]}' --header "Content-Type: application/json" http://localhost:8545```
```

Comment:
What does an empty pubSub topic signify here? Shouldn't this be the default pubsub topic?

Reply:
empty means "all" right?

Reply:
ah right